### DISCUSSION TITLE
minio RELEASE.2022-04-08T19-44-35Z   one host one disk is faster than one host four disks

### DISCUSSION THREAD

**Discussion by queguan** [minio/minio#14720] [Q&A] [answered]
hello ,
     I  setup minio server on an (48 core ; 256G memory ; sata disk ;10000Mbs network ).  I tested and found one server one disk`s throughput is four times  than one server four disks when on read/write big files .Can anyone help explain why? Thanks for the reply.

**Comment by harshavardhana** [comment, marked as answer]
You need to check your disk @queguan
